new-starter-guide/new-starter-to-origami.md

```diff
@@ -0,0 +1,123 @@
+
+### FT.com
+
+Check your email to see if you’ve already been registered for the free subscription to the [ft.com](https://www.ft.com/). If not, sign-up for an account on ft.com then ask `corporate.support@ft.com` to upgrade it to a free subscription.
+
+## Development tools
+
+### Environment setup
+
+Before you start installing anything, it is a good idea to do this (with admin privileges) to ensure that <abbr title="Node Package Manager">npm</abbr> is given permissions to install globally on your machine.
+
+```
+chown -R $USER /usr/local
+```
+
+Almost all Origami applications rely on the following tools to be installed globally on development machines. Please ensure both of these are installed successfully onto your development environment first.
+
+#### Install the following:
+
+- [Node.js](https://nodejs.org/)
+  - We recommend installing a [Node Version Manager](https://github.com/creationix/nvm) to manage different local node versions.
+- [Git](https://git-scm.com/)
+- [Heroku Toolbelt](https://toolbelt.heroku.com/)
+- [Bower](http://bower.io/)
+- [Ruby](http://www.ruby-lang.org/)
+  - We recommend installing a Ruby Version Manager to manage different ruby versions. There are two choices here. You can go either [Ruby Version Manager](https://rvm.io/) or [Rbenv](https://github.com/rbenv/rbenv).
+  - This is for the Ruby SCSS-Lint gem which is a part of <abbr title="Origami Build Tools">OBT</abbr>.
+
+#### Work with our ecosystem of components
+
+You will need to point Bower at the Origami registry. Running the following in a <abbr title="Command Line Interface">CLI</abbr> will do this for you.
+
+```
+[ -e ~/.bowerrc ] || echo '{ "registry": { "search": [ "http://registry.origami.ft.com", "https://bower.herokuapp.com" ] } }' > ~/.bowerrc
+```
+
+You can join all the Origami applications on Heroku by checking out this [shared Google Sheets](https://docs.google.com/a/ft.com/spreadsheets/d/1xk1tyn60ZCmLk1I39Dot-08c9pBJeeX3g9MDENDqjKk/edit?usp=drive_web). You will need one other person from the Origami team to add you.
+
+
+#### Issues with installing/getting access
+
+If you experience any problems, then please take a look at Next's [troubleshooting guide](http://financial-times.github.io/next/docs/developer-guide/troubleshooting/). If your problem is not included in the troubleshooting list and you manage to solve it, then please add it to the list.
+
+### Peer review and pull requests
+
+With all the work that you would produce for Origami and when you reach the stage of doing a pull request, ask someone within the team for a review of your work. If it gets the thumbs up from the reviewer then you're free to merge it into the master branch of the repository. It also depends on the size of the work that was produced, if it's quite substantial then it would require more people to look over the work before giving the all clear to merge.
```

Comment:
I think we could re-write this a bit. Maybe something like:

All code in Origami is written by one person and peer reviewed by one or more people using git branches and merging.
Here's how it works
1. Branch off of master: `git branch my-great-feature`
2. [some coding / commiting happens in the my-great-feature branch\
3. Push those changes to github / stash: `git push origin my-great-feature-branch`
4. Repeat 2 and 3 as much as you like
5. When you're ready, open a pull request. Tag people you'd like to review the request.
6. Respond to review comments
7. Either a reviewer will merge your request, or they will give you a thumbs-up for you to merge.